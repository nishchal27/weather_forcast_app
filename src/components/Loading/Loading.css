.Loading {
  width: 3em;
  height: 3em;
  display: flex;
	transition: color 0.25s;
	border: solid 3px #eee;
	border-bottom-color: var(--primary-color);
	border-radius: 50%;
	animation: 1s linear infinite spin;
}

@keyframes spin {
  0% {
    transform: translate3d(-50%, -50%, 0) rotate(0deg);
  }
  100% {
    transform: translate3d(-50%, -50%, 0) rotate(360deg);
  }
}
