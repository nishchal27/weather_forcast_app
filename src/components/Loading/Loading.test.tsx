import React from 'react';
import { shallow } from 'enzyme';

import Loading from '.';

describe('<Loading />', () => {
  it('renders without crashing', () => {
    const wrapper = shallow(<Loading />);

    expect(wrapper).toMatchSnapshot();
  });
});
