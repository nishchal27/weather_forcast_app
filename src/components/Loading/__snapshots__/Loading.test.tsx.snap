// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Loading /> renders without crashing 1`] = `
<div
  className="Loading"
/>
`;
